components/countdown-text-area/src/index.js

```diff
@@ -86,7 +86,7 @@ const CountdownTextArea = (props) => {
   const over = showRemaining && remaining <= 0;
 
   return (
-    <StyledTextArea over={over}>
+    <StyledTextArea over={over} {...props}>
```

Comment:
i'm not sure this is right

feels like for here that we may wish to extract and set `id` and `className` only - the other props are only relevant for the `StyledTextAreaField` component

alternatively we just leave this as-is - i'm not sure of the advantage of setting any props on the container here

Reply:
Yes, would mainly just be to handle props like `className` passed down from HOCs. Maybe we should leave this as is for now, now it's done, but can can reconsider later.

Reply:
by "as-is" i meant before this edit was made :grin:

we can leave this in, as passing down extraneous properties isn't necessarily a biggie, but i'd be more comfortable if we at least put a note against this in the code to reconsider what we're doing here

Reply:
Yes, I know what you mean - we can't explicitly see what is being sent down. I'll raise an issue, so it's logged, for us to consider extracting and sending down only specific props in the future.